src/rpdk/core/contract/resource_client.py

```diff
@@ -168,12 +176,14 @@ def _update_schema(self, schema):
         self.write_only_paths = self._properties_to_paths("writeOnlyProperties")
         self.create_only_paths = self._properties_to_paths("createOnlyProperties")
         self.properties_without_insertion_order = self.get_metadata()
-
+        self.property_transform_keys = self._properties_to_paths("propertyTransform")
+        self.property_transform = self._schema.get("propertyTransform")
         additional_identifiers = self._schema.get("additionalIdentifiers", [])
         self._additional_identifiers_paths = [
             {fragment_decode(prop, prefix="") for prop in identifier}
             for identifier in additional_identifiers
         ]
+        self.jinja_template = self.env.get_template("transformation.template")
```

Comment:
nit: remove jinja from name and replace with something like "transformation" to indicate what the template actually contains and not the library used to transform it.